docs/accessibility/q-a-testing.md

```diff
@@ -219,4 +220,96 @@ One neat feature included within Firefox's accessibility inspector is the abilit
+
+    import { RhTag } from '@rhds/elements/rh-tag/rh-tag.js';
+
+    describe('when the element loads', function() {
+      let element: RhTag;
+      beforeEach(async function() {
+        element = await createFixture<RhTag>(html`
+          <rh-tag>Default tag</rh-tag>
+        `);
+        await element.updateComplete;
+      });
+
+      it('should be accessible', async function() {
+        await expect(element)
+            .to.be.accessible();
+      });
+    });
+    ```
+1. Run this test in your terminal by typing: `npm run test -- -- elements/rh-tag/test/rh-tag.spec.ts`
+1. You should see "1/1 test files | 1 passed, 0 failed" output to your terminal.
+1. Congrats, you just implemented aXe a11y tests into your testing pipeline! 🎉
+
+In this test, we called the Chai A11y aXe module and ran it against `<rh-tag>` to ensure `<rh-tag>` didn't fail any automated accessibility tests as reported by aXe. Every component in the Red Hat Design System should call and use Chai A11y aXe to continuously verify accessibility in RHDS components.
+
+If you want to see what a failed test might look like, add something to `<rh-tag>` that aXe would flag for failing an accessibility test—like `tabindex="1234"`. Then, re-run the test (step 4). Your terminal will show "Error: Accessibility Violations" and provide more information about what failed and how to fix it.
+
+##### a11ySnapshot
+
+Web Test Runner's [`a11ySnapshot` command](https://modern-web.dev/docs/test-runner/commands/#accessibility-snapshot) requests a snapshot of the accessibility tree built in the browser. Web component authors can use `a11ySnapshot` to verify accessibility properties in unit tests.
+
+Here's how to integrate `a11ySnapshot` with `<rh-skip-link>`:
+
+1. Follow the instructions to [Install the project](https://ux.redhat.com/get-started/developers/contributing/).
+1. Open `elements/rh-skip-link/test/rh-skip-link.spec.ts`.
+    * Note: there are already tests for `<rh-skip-link>`. For this exercise, let's pretend this file is blank. Temporarily delete its contents.
+1. Paste the following into `rh-skip-link.spec.ts`:
+    ```ts
+    import { expect, html } from '@open-wc/testing';
+    import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+    import { a11ySnapshot } from '@patternfly/pfe-tools/test/a11y-snapshot.js';
+    import { RhSkipLink } from '@rhds/elements/rh-skip-link/rh-skip-link.js';
+
+    describe('<rh-skip-link>', function() {
+      describe('when the element loads', function() {
+        let element: RhSkipLink;
+        beforeEach(async function() {
+          element = await createFixture<RhSkipLink>(html`
+            <rh-skip-link>
+              <a href="#main-content">
+                Skip to main content
+              </a>
+            </rh-skip-link>
+          `);
+          await element.updateComplete;
+        });
+
+        it('should have a link with the name, "Skip to main content"', async function() {
```

Comment:
can we use the chai helpers here instead of deep assertions?

see https://bennypowers.dev/decks/semantic-component-testing/#slide-25/step-3

Reply:
Great suggestion!

Addressed in [5ed9a66](https://github.com/RedHat-UX/red-hat-design-system/pull/1903/commits/5ed9a66afdc28a0483adc8f8d3bdefc261b6f1f5). I also added some copy + a link to PFE tools to reference the custom Chai helpers to point people in the right direction. PTAL and LMK what you think.